packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -1300,6 +1349,68 @@ describe('StripeHelper', () => {
       });
       assert.deepEqual(actual, expected);
     });
+
+    it('retrieves details when previewInvoice rejects', async () => {
```

Comment:
Very possible that it was just me but I found this test title confusing.  I was trying to understand _what_ is retrieving coupon details when previewInvoice rejects, when there's no retrieving call of any sort after previewInvoice in the function.

Reply:
I updated the text a little that should hopefully clarify it a bit. 

These are all `retrieveCouponDetails` tests. That test specifically should still return a result even though the method called by it, previewInvoice, rejects.